### ISSUE TITLE
Verify Sweatcoin performance with flat storage

### ISSUE THREAD

**Issue by jakmeier** [near/nearcore#7676]
Once #7327 is done, the IO performance characteristics will change drastically. We want to proactively measure the most largest IO user on chain today, which is Sweatcoin. And we want to remove some hacky code we introduced recently, see #7637.

Rough plan of action for this issue:
- Figure out how many DB requests will SWEAT load be on paper, if flat storage is enabled?
- Compare expected time to serve SWEAT requests vs gas costs.
- Can we confidently remove the argument prefetcher once we have flat storage enabled?

**Comment by andrei-near** [comment]
@jakmeier believe this can be closed?

**Comment by jakmeier** [comment]
yeah this is no longer relevant
